specification/hugr.md

```diff
@@ -893,20 +893,20 @@ resources:
     # Hence, no signature block => will look up a compute_signature in registry.
   - name: GraphOp
     description: "Involves running an argument Graph. E.g. run it some variable number of times."
-    args:
+    params:
       - r: ResourceSet
```

Comment:
Slightly off-topic, but I don't think `ResourceSet` as a param is valid

Reply:
raise issue?